bundles/binding/org.openhab.binding.plclogo/src/main/java/Moka7/IntByRef.java

```diff
@@ -0,0 +1,34 @@
+/**
+ * Copyright (c) 2010-2016 by the respective copyright holders.
+ *
+ * Moka7 librray is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Library or Lesser General Public License
+ * version 3.0 (LGPLv3) as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ */
```

Comment:
@watou I have fixed minor project things. What will be right approach of including Moka7 library?

According to actual version of their sources - they have only copyright placeholder at beginning of source files. In reference document the claim that claim that Moka7 distributed under GNU Library or LGPLv3 license.